spec/text-editor-presenter-spec.coffee

```diff
@@ -1338,7 +1340,9 @@ describe "TextEditorPresenter", ->
               blockDecoration3 = addBlockDecorationBeforeScreenRow(7)
               blockDecoration4 = null
 
-              waitsForStateToUpdate presenter, blockDecoration4 = addBlockDecorationAfterScreenRow(7)
```

Comment:
This lil' block decoration was being passed as the callback argument to `waitsForStateToUpdate`, which was silently accepting it because of the `fn?()` syntax.